pipelines/official_build.yml

```diff
@@ -0,0 +1,32 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+stages:
```

Comment:
what is the trigger?

Reply:
trigger can be overide by UI config. And I have tried yaml format trigger: [schedules](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml), which not work as expected.
So I perfer [UI config.](https://dev.azure.com/sonicswitch/build/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=33&view=Tab_Triggers)